src/families/filecoin/test-dataset.ts

```diff
@@ -11,7 +11,7 @@ import { fromTransactionRaw } from "../filecoin/transaction";
 import { CurrenciesData } from "../../types";
 
 const SEED_IDENTIFIER = "f1zx43cf6qb6rd5e4okl7lexnjumxe5toqj6vtr3i";
-const ACCOUNT_1 = "t1d2xrzcslx7xlbbylc5c3d5lvandqw4iwl6epxba";
+const ACCOUNT_1 = "f1zx43cf6qb6rd5e4okl7lexnjumxe5toqj6vtr3i";
```

Comment:
Just checking, is this account frozen, i.e will it be never used again?
We need to use frozen accounts here so that we don't need to regenerate the snapshots all the time.

Reply:
I didn't know that! Let me change it then so we can leave an account frozen!

Reply:
The account now is frozen!